data/mods/MMA/techniques.json

```diff
@@ -60,7 +60,7 @@
     "min_melee": 5,
     "unarmed_allowed": true,
     "melee_allowed": true,
-    "messages": [ "You kick %s and spin around", "<npcname> kicks %s and spins around" ],
+    "messages": [ "The %s tries to grab you, but you kick free and spin around!", "The %s tries to grab <npcname>, but they kick free and spin around!" ]
```

Comment:
```suggestion
    "messages": [ "The %s tries to grab you, but you kick free and spin around!", "The %s tries to grab <npcname>, but they kick free and spin around!" ],
```

Reply:
Good catch!

Reply:
Easy to catch. It shines bright like a.. zombie blood.
![github-json](https://user-images.githubusercontent.com/1931904/58750296-8e173f00-84a1-11e9-819a-4238de841f87.png)